javascript/utils/time.ts

```diff
@@ -0,0 +1,3 @@
+export function delay(ms: number) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
```

Comment:
super nit:
```suggestion
export async function delay(ms: number) {
  await new Promise(resolve => setTimeout(resolve, ms));
}
```
so that delay() is on the traceback when someone is awaiting a delay. Again, super nit.